.travis.yml

```diff
@@ -15,13 +15,29 @@ language: python
 matrix:
   include:
     - python: 2.7
-      env: TOXENV=py27
+      env: TOXENV=py27-static
+    - python: 2.7
+      if: type = cron
+      env: TOXENV=py27-dynamic
+    - python: 2.7
+      if: type = cron
+      env: TOXENV=py27-upgrade
+    - python: 3.6
+      if: type = cron
+      env: TOXENV=py36-static
     - python: 3.6
-      env: TOXENV=py36-default
+      env: TOXENV=py36-dynamic
     - python: 3.6
+      if: type = cron
       env: TOXENV=py36-upgrade
     - python: 3.7
-      env: TOXENV=py37
+      if: type = cron
+      env: TOXENV=py37-static
+    - python: 3.7
+      if: type = cron
+      env: TOXENV=py37-dynamic
+    - python: 3.7
+      env: TOXENV=py37-upgrade
 before_install:
   - docker pull quay.io/pulp/pulp-ci-c7:3.0.0
```

Comment:
The docker pulls are only needed for the upgrade scenarios. They probably take a few to several mins.

This fell through the cracks and is originally a result of a prior PR.